packages/components/src/list/experimental-list-item.tsx

```diff
@@ -0,0 +1,78 @@
+/**
+ * External dependencies
+ */
+import { CSSTransition } from 'react-transition-group';
+import classnames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import { handleKeyDown } from './list-item';
+
+type CSSTransitionProps = {
+	in: boolean;
+	exit: boolean;
+	enter: boolean;
+	onExited: () => void;
+};
+
+type ListItemProps = {
+	// control whether to display padding on list item or not.
+	disableGutters: boolean;
```

Comment:
@samueljseay `disableGutters` should also be optional here, given it defaults to `false`.

Reply:
I suppose the same for the `CssTransitionProps`

Reply:
Thanks will fix the optionals

Reply:
Fixed here https://github.com/woocommerce/woocommerce-admin/pull/6787/commits/6d47d9a6fde9cf27811c6a72897e82817837b759